bindata/network/ovn-kubernetes/common/008-script-lib.yaml

```diff
@@ -310,10 +310,6 @@ data:
       if ! retry 60 "inactivity-probe" "ovn-nbctl -t 5 --inactivity-probe={{.OVN_NB_INACTIVITY_PROBE}} set-connection punix:${nbdb_sock}"; then
         exit 1
       fi
-      # set trim-on-compaction
-      if ! retry 60 "trim-on-compaction" "ovn-appctl -t ${nbdb_ctl} --timeout=5 ovsdb-server/memory-trim-on-compaction on"; then
```

Comment:
based on what Ric wrote here: https://issues.redhat.com/browse/OCPBUGS-237?focusedId=21388389&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-21388389 cc @ricky-rav please review this PR, this change lgtm
However now how do we verify if in RHEL9 this is enabled by default in latest OVS? @arghosh93 did you check OVS to ensure that is indeed the case?

Reply:
Ah I guess this is it: https://github.com/openvswitch/ovs/commit/e773140ec3f6d296e4a3877d709fb26fb51bc6ee

Reply:
so the database compaction is enabled by default starting from ovs 3.0.0 and we've been using ovs 3.0.0+ since openshift 4.13, where we moved to RHEL9:
https://docs.google.com/document/d/1yWZJaVBs7z1KPs4ZKdz7CIBVDMS3bLMymW_wBYMCqKw/edit